### PR TITLE
Convert ProfileView Component from ANT Design to Material UI

### PR DESCRIPTION
**Pull request by piyumaldk**
In order to do so, I replaced all the AntD components with Material UI core, core/style, and icon components.

## Description
Other than replacing those components, I added a MuiTheme to remove shadows from the card and add a linear loading component. I also had to design a simple svg file as an alternative image if the user doesn't have an image. 
Svg file can be found in the src/asset/images path and the file name is "no-image-available.svg".

## Related Issue
#87 _Convert ProfileView Component from ANT Design to Material UI_

## Motivation and Context
It is needed to remove all the Antd from the CodeLabz project, so this is one component. 

## How Has This Been Tested?
Manually tested the component.

- No warnings were generated in the console of the browser and the command prompt where the project was run. 
- No errors were generated in the console of the browser and the command prompt where the project was run. 
- Everything worked same as before.
- Merged the Master branch of the project and fixed conflicts.
- After the merging, retested the component. 

## Screenshots (In case of UI changes):
| Before (Antd)  | After (Material UI) |
| ------------- | ------------- |
|before|after|

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/components/Dashboard/index.js [hunk 1]
```diff
@@ -6,14 +6,14 @@ import {
...
-  setUpInitialData
+  setUpInitialData,
 } from "../../store/actions";
 import {
   GlobalOutlined,
   UserAddOutlined,
   AppstoreAddOutlined,
   AppstoreOutlined,
-  IeOutlined
+  IeOutlined,
```
Comment (Niweera) [inline src/components/Dashboard/index.js:16]: Follow Material-UI import guidelines as mentioned in https://github.com/scorelab/Codelabz/pull/75#discussion_r638543643.

File: src/components/Home/index.js [hunk 2]
```diff
@@ -1,16 +1,24 @@
+import {
+  Button,
+  Card,
+  CardActionArea,
+  CardActions,
+  CardContent,
+  CardMedia,
+  Grid,
+  Typography,
+} from "@material-ui/core";
```
Comment (Niweera) [inline src/components/Home/index.js:10]: Please change according to Material-UI import guidelines as mentioned in https://github.com/scorelab/Codelabz/pull/75#discussion_r638543643.

**Review by Niweera — CHANGES_REQUESTED** [review]
Please do the necessary changes as mentioned in the comments.

File: src/components/Home/index.js (hunk 2 above)
Comment (piyumaldk) [reply src/components/Home/index.js]:
```
import Button from "@material-ui/core/Button";
import Card from "@material-ui/core/Card";
import CardActionArea from "@material-ui/core/CardActionArea";
import CardActions from "@material-ui/core/CardActions";
import CardContent from "@material-ui/core/CardContent";
import CardMedia from "@material-ui/core/CardMedia";
import Grid from "@material-ui/core/Grid";
import Typography from "@material-ui/core/Typography";
```

Done

**Review by Niweera — APPROVED** [review]

**Pull request merged**